assets/js/script.js

```diff
@@ -3,103 +3,124 @@ M.AutoInit();
 
 // DOM VARIABLES GO HERE
 const wineSelectorEl = document.querySelector('#wineSelector'); 
-const mainEl = document.querySelector('#main');
+const contentRow = document.querySelector('#contentRow');
 const headerEl = document.querySelector('#header');
 const foodSelectorEl = document.querySelector('#foodSelector');
 const randomPairingEl = document.querySelector('#randomPairing');
 const aboutUsEl = document.querySelector('#about-us');
-const apiKey = '2e045af459ca42fda601b67a39611082';
+const apiKey = 'f9e4f37f62de4dba85137360011a63c2';
 
 const ourNames = ['Tim Weyel', 'Shy Gois', 'Leah Russell', 'Sydney Walcoff', 'Carlos Vadillo'];
 const ourTitles = ['Director of HTML', 'Data Courier', 'Github Cat Wrangler', 'Chief Mischief Officer (CMO)', 'Unpaid Intern'];
 
 //create a function to remove all html elements except the footer
-const removeExistingElems = () => {
+const removeEl = () => {
 	headerEl.innerHTML = '';
-	mainEl.innerHTML = '';
+	contentRow.innerHTML = '';
 };
 
+const recipePageConstructor = () => {
+	const recipeCardDiv = document.createElement("div");
+	const recipeUl = document.createElement("ul");
+	recipeUl.setAttribute("class", "collapsible");
+	recipeUl.setAttribute("id", "recipeUl");
+	recipeCardDiv.classList.add("col", "s12", "m7");
+	recipeCardDiv.append(recipeUl);
+	contentRow.append(recipeCardDiv);
+};
+
+
 const listenerHandler = el => {
 	el.addEventListener('change', function() {
-		const title = this.options[this.selectedIndex].text;
+		const selectedOption = this.options[this.selectedIndex].text;
 		const dataType = this.options[0].text.split(' ')[0];
-		console.log(title, dataType);
-		fetchData(dataType, title);
-		removeExistingElems();
+		validateDropdownType(dataType, selectedOption);
+		removeEl();
+		contentRow.setAttribute("class", "container row");
 		// these will go inside the fetch function because that's where we'll receive input for ratings, descr, and recipes
-		// infoCardGenerator(title, info, rating);
+		// infoCardGenerator(selectedOption, info, rating);
 		// recipeCardGenerator(firstProtein, secondProtein(if applicable),...n);
 	});	
 };
 
 // fetch data
-const fetchData = (dataType, title) => {
+const validateDropdownType = (dataType, selectedOption) => {
```

Comment:
noice. totally wanted to do this last night during the surprise me PR, but didn't want to mess anyone up.